CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Removed
 ### Fixed
 
+### Added
+- Added `Engine.getWorldBounds`
+- Added test for `Engine.getWorldBounds`
```

Comment:
We don't neet to mention that we added tests in the changelog, let's remove this line